docs/source/index.rst

```diff
@@ -1,4 +1,22 @@
 Welcome to QMAP's documentation!
 ================================
 
-At the moment, this documentation is just blank. Expect this to change in the near future.
+QMAP is a tool for quantum circuit mapping developed as part of the *Munich Quantum Toolkit* (*MQT*) [#]_ by the `Chair for Design Automation <https://www.cda.cit.tum.de/>`_ at the `Technical University of Munich <https://www.tum.de>`_. It builds upon `our quantum functionality representation (QFR) <https://github.com/cda-tum/qfr>`_.
+
+If you have any questions, feel free to contact us via `quantum.cda@xcit.tum.de <mailto:quantum.cda@xcit.tum.de>`_ or by creating an issue on `GitHub <https://github.com/cda-tum/qmap/issues>`_.
+
+----
+
+.. toctree::
+   :maxdepth: 3
+
+   Installation
+   Library
+   Publications
+   Mapping
```

Comment:
```suggestion
   Mapping
   Installation
   Library
   Publications
```
I am in favor of putting the "content" first and then talk about installation and the library in particular.

Reply:
An oversight on my part. That was the intention anyway.